com.unity.render-pipelines.core/ShaderLibrary/UnityInstancing.hlsl

```diff
@@ -239,6 +239,11 @@
 
     #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/UnityDOTSInstancing.hlsl"
 
+    #if defined(UNITY_SETUP_INSTANCE_ID)
+        #undef UNITY_SETUP_INSTANCE_ID
+        #define UNITY_SETUP_INSTANCE_ID(input) { DEFAULT_UNITY_SETUP_INSTANCE_ID(input); SetupDOTSVisibleInstancingData(); }
```

Comment:
This should probably be ifdeffed so that the `SetupDOTSVisibleInstancingData` is only called in DOTS_INSTANCING_ON variants.

Reply:
This is already in a block of code that has UNITY_DOTS_INSTANCING_ENABLED. But Ill add here here just to be more expressive :)